com.unity.render-pipelines.core/Runtime/Utilities/Blitter.cs

```diff
@@ -235,7 +235,8 @@ public static void BlitColorAndDepth(CommandBuffer cmd, Texture sourceColor, Ren
         public static void BlitTexture(CommandBuffer cmd, RTHandle source, Vector4 scaleBias, Material material, int pass)
         {
             s_PropertyBlock.SetVector(BlitShaderIDs._BlitScaleBias, scaleBias);
-            s_PropertyBlock.SetTexture(BlitShaderIDs._BlitTexture, source);
+            if (source?.rt != null)
```

Comment:
Change is in core.Blitter, so needs a review from HDRP too. And at least some basic DX11 sanity check testing.

Reply:
It might be possible to revert this particular change. It could be leftover from my migration.

However, the check for activated scaling requires HDRP review.